styles/styles.css

```
@@ -15,61 +15,67 @@
 }
 
 body {
-  font-family: "PT-Sants";
+  font-family: "PT-Sants", sans-serif;
```

```suggestion
  font-family: "PT-Sants", "Aria", sans-serif;
```
должен быть ещё безопасный шрифт